c/dec/decode.c

```diff
@@ -1420,7 +1421,7 @@ SkipMetadataBlock(BrotliDecoderState* s) {
   /* Drain accumulator. */
   if (BrotliGetAvailableBits(br) >= 8) {
     uint8_t buffer[8];
-    int nbytes = (int)(BrotliGetAvailableBits(br)) >> 3;
+    nbytes = (int)(BrotliGetAvailableBits(br)) >> 3;
```

Comment:
I'm just passing by, but you don't need to move this declaration, as it's already at the start of it's own block.

Reply:
Indeed. But by moving the other declaration, this one shadows it.

It doesn't affect the correctness of the code, but it's "bad form" and some static analysers might whinge about it.